packages/blade/src/storybook-recipes/AccessibilityLayer/AccessibilityLayer.native.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Pressable, Text, View } from 'react-native';
+import makeAccessible from '../../utils/makeAccessible';
+import { CloseIcon, EyeIcon } from '../../components/Icons';
+import { getCheckboxA11yProps } from './getCheckboxA11yProps';
+
+const CheckboxItem: React.FC<{ label: string; checked?: boolean; disabled?: boolean }> = ({
+  label,
+  checked,
+  disabled,
+}) => {
+  const [isChecked, setChecked] = React.useState(checked);
+  const checkboxA11yProps = getCheckboxA11yProps({ isDisabled: disabled, isChecked });
```

Comment:
nit: give some mehnat to you finger sand make this name accessible :P 

```suggestion
  const checkboxAccessibilityProps = getCheckboxAccessibilityProps({ isDisabled: disabled, isChecked });
```